### PR TITLE
move webpage to Gatsby

### PR DESCRIPTION
**Pull request by kwypchlo**
- switched to Gatsby https://www.gatsbyjs.org/ to improve first load, SEO and generated icons
- inlined some svgs
- fixed a11y issue with code examples buttons not being accessible
- refactored the logo to be a square so the icons are generated properly
- used light build of syntax highlighter to shave off unnecessary bytes
- removed leftover packages directory

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/browser-apis/
+ */
+
+// You can delete this file if you're not using it
```
Comment (peterjanbrone) [inline gatsby-browser.js]: Q: are we using it?

File: gatsby-node.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/node-apis/
+ */
+
+// You can delete this file if you're not using it
```
Comment (peterjanbrone) [inline gatsby-node.js]: I understand these are just the defaults you left in, just wondering if we'll ever use them

File: gatsby-ssr.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/ssr-apis/
+ */
+
+// You can delete this file if you're not using it
```
Comment (peterjanbrone) [inline gatsby-ssr.js]: Especially this one..

File: package.json [hunk 4]
```diff
@@ -1,50 +1,57 @@
...
     "react-reveal": "^1.2.2",
-    "react-scripts": "^3.4.0",
     "react-syntax-highlighter": "^12.2.1",
     "react-visibility-sensor": "^5.1.1",
     "shortid": "^2.2.15"
   },
   "devDependencies": {
     "prettier": "^1.19.1"
   },
+  "keywords": [
```
Comment (peterjanbrone) [inline package.json]: perhaps add "decentralized" and "cloud storage"

File: LICENSE [hunk 5]
```diff
@@ -1,4 +1,4 @@
-MIT License
+The MIT License (MIT)
```
Comment (peterjanbrone) [inline LICENSE]: Not that it matters but I think we can drop `(MIT)`

File: src/components/seo.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        {
+          name: `twitter:description`,
+          content: site.siteMetadata.description
+        }
+      ].concat(meta)}
+    />
+  );
+}
+
+SEO.defaultProps = {
```
Comment (peterjanbrone) [inline src/components/seo.js]: These defaults are overwritten by the values returned by the graphql query, right?

File: package.json [hunk 7]
```diff
@@ -1,50 +1,57 @@
...
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "format": "prettier --write \"**/*.{js,jsx,json,md}\""
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "yarn run develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
```
Comment (peterjanbrone) [inline package.json]: Perhaps delete this script ^^

File: gatsby-browser.js (hunk 1 above)
Comment (kwypchlo) [reply gatsby-browser.js]: Yeah I think it's safe to delete this now, we are not going to use it any time soon here.

File: gatsby-node.js (hunk 2 above)
Comment (kwypchlo) [reply gatsby-node.js]: Yeah I think it's safe to delete this now, we are not going to use it any time soon here.

File: gatsby-ssr.js (hunk 3 above)
Comment (kwypchlo) [reply gatsby-ssr.js]: Yeah I think it's safe to delete this now, we are not going to use it any time soon here.

File: package.json (hunk 4 above)
Comment (kwypchlo) [reply package.json]: Good catch.

File: LICENSE (hunk 5 above)
Comment (kwypchlo) [reply LICENSE]: It's just the default readme template from Gatsby, I don't have preference, will revert it to what it's been.

File: src/components/seo.js (hunk 6 above)
Comment (kwypchlo) [reply src/components/seo.js]: Not really, it's just the default language and empty meta here. GraphQL provides title, description and author.

File: package.json (hunk 7 above)
Comment (kwypchlo) [reply package.json]: uhh

**Pull request merged**